### PR TITLE
task1-Ferhat

### PR DESCRIPTION
**Pull request by FerhatPeynirci**

### CODE REVIEW

File: task1-gradient-descent/GradientDescent_Ferhat.ipynb [hunk 1]
```diff
@@ -0,0 +1,1626 @@
+{
```
Comment (bkavlak) [inline task1-gradient-descent/GradientDescent_Ferhat.ipynb:1]: <p>It's good to go deep and calculate all math in your code in the early stages to understand the theory. In practice, you are not gonna calculate this way. You find a function do it for your like here:</p><p> </p><p>https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (bkavlak) [inline task1-gradient-descent/GradientDescent_Ferhat.ipynb:1]: <p>See this video for the logic behind:</p><p> </p><p>https://www.youtube.com/watch?v=CbsegCURCd0</p><p> </p><p>The answer is correct but I'm not sure if the gist behind is understood here. This is a good explanation:</p><p>https://stackoverflow.com/questions/49147774/what-is-random-state-in-sklearn-model-selection-train-test-split-example#:~:text=If%20you%20don%27t,and%20test%20datasets.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>
